Goobi/src/de/sub/goobi/export/dms/AutomaticDmsExportWithoutHibernate.java

```diff
@@ -138,9 +138,6 @@ public boolean startExport(ProcessObject process) throws DAOException, IOExcepti
 				break;
 
 			case METS_AND_RDF:
```

Comment:
If somebody still use RDF as store and / or export format then he should get a proper deprecation message. One or two versions later this format should be removed and not supported formats should throw an error message in the gui.